plugins/dss/hooks.js

```diff
@@ -71,23 +76,33 @@ function getContainerMounts(containers, volumeMountIndex, volumeMounts) {
 }
 
 function VolumeMountTypesSelect(props) {
+  const { volumes, index } = props;
+
   return (
-    <FieldSelect
-      name={`volumeMounts.${props.index}.type`}
-      value={props.volumes.type}
+    <Select
+      name={`volumeMounts.${index}.type`}
+      value={volumes.type}
+      placeholder="Select ..."
     >
-      <option value="">Select...</option>
-      <option value={VolumeConstants.type.host}>Host Volume</option>
-      <option value={VolumeConstants.type.ephemeral}>
-        Ephemeral Volume
-      </option>
-      <option value={VolumeConstants.type.localPersistent}>
-        Local Persistent Volume
-      </option>
-      <option value={VolumeConstants.type.dss}>
-        DC/OS Storage Service
-      </option>
-    </FieldSelect>
+      {Object.keys(VolumeDefinitions)
+        .filter(type => !excludedTypesMultiContainer.includes(type))
+        .map((type, index) => {
```

Comment:
Sorry I didn't mention this earlier, but both maps in this file are exactly the same, which for me means we could move it into a separate function. I don't want to slow us down here, maybe we can create a small jira issue to tackle somewhen on the end of the sprint when there is time left 👍

Reply:
To be honest, basically the entire function is pretty similar, I guess we could abstract that duplication entirely away 🤔

Reply:
I agree, since they are entirely the same, it would make sense to extract or reuse one of them.

Reply:
I'll go ahead and do this.

Reply:
Refactored the function.